<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindBar</class>
 <widget class="QWidget" name="FindBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>33</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="window-close">
       <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblFind">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtFindText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnPrevious">
     <property name="text">
      <string>Previous</string>
     </property>
     <property name="icon">
      <iconset theme="go-previous">
       <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnNext">
     <property name="text">
      <string>Next</string>
     </property>
     <property name="icon">
      <iconset theme="go-next">
       <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnHighlight">
     <property name="text">
      <string>Highlight All</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkMatchCase">
     <property name="text">
      <string>Match Case</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
